app/src/main/java/net/skyscanner/backpack/demo/data/BpkBarChartData.kt

```diff
@@ -80,14 +80,24 @@ object BpkBarChartData {
 
   fun createBar(
     date: LocalDate,
-    locale: Locale = Locale.getDefault(),
+    resources: Resources,
     values: BpkBarChartModel.Values? = null,
-  ): BpkBarChartModel.Item =
-    BpkBarChartModel.Item(
+  ): BpkBarChartModel.Item {
+    val locale = resources.configuration.locales.get(0)
+    val dayOfMonth = date.dayOfMonth.toString()
+    val dayOfWeek = date.dayOfWeek.getDisplayName(TextStyle.SHORT, locale)
+    val fullDayOfWeek = date.dayOfWeek.getDisplayName(TextStyle.FULL, locale)
+    return BpkBarChartModel.Item(
       key = date,
-      title = date.dayOfWeek.getDisplayName(TextStyle.SHORT, locale),
-      subtitle = date.dayOfMonth.toString(),
+      title = dayOfWeek,
+      subtitle = dayOfMonth,
       group = date.month.getDisplayName(TextStyle.FULL, locale),
       values = values,
+      accessibilityLabel = if (values != null) {
+        resources.getString(R.string.bar_chart_accessibility_label, fullDayOfWeek, dayOfMonth, values.text)
```

Comment:
rather than building the date string ourselves we can use the same logic we apply in calendar to build a date

Reply:
I think it's an overkill for a sample app. It's important to show the principle, but using resources, currency, and date formatters make the code harder to read (which is most important for a sample) and maintain